Swift 4 .podspec missing Tools/curry.swift` file

### Name and description
RxSwiftExt.podspec for Swift 4 branch missing `Tools/curry.swift` file in it's `source_files` parameter
### Motivation for inclusion
I'm getting Error in Xcode 9 when adding RxSwiftExt to my Podfile with specified branch.
### Example of use
My Podfile looks like this:
pod 'RxSwiftExt',:git => 'https://github.com/RxSwiftCommunity/RxSwiftExt.git', :branch => 'swift4.0'
After running `pod update` command I'm getting errors in Xcode:
```
.../Pods/RxSwiftExt/Source/RxSwift/fromAsync.swift:31:49: error: use of unresolved identifier 'curry'
```

--- a/RxSwiftExt.podspec
+++ b/RxSwiftExt.podspec
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/RxSwiftCommunity/RxSwiftExt.git", :tag => s.version }
 
   s.subspec "Core" do |cs|
-    cs.source_files  = "Source/RxSwift"
+    cs.source_files  = "Source/RxSwift", "Source/Tools"
     cs.frameworks  = "Foundation"
     cs.dependency "RxSwift", '~> 4.0.0-beta.0'
   end

